modules/issue_tracker/php/attachment.class.inc

```diff
@@ -148,7 +148,7 @@ class Attachment extends \NDB_Page implements ETagCalculator
             $DB      = $factory->database();
             $DB->update(
                 'issues_attachments',
-                ['deleted' => 1],
+                ['deleted' => 1, 'date_deleted' => date('Y-m-d H:i:s')],
```

Comment:
I wonder is deleted is still necessary? Having date_deleted <> NULL is sufficient to tell if an attachment is deleted.

Reply:
@laemtl It is necessary otherwise will through an error

Reply:
@laemtl Yes we still need  the `date_deleted` column to display the `date of delation`.

Reply:
@laemtl Yes we do still need `deleted` column in the database table.

Reply:
This is used only once:

`if (parseInt(item.deleted) === 1) {`

which can become

`if (item.date_deleted) {`

Reply:
I would second Laetitia's thought here!

Reply:
I agree.